examples/src/main/java/ai/djl/examples/inference/face/FaceDetectionTranslator.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package ai.djl.examples.inference.face;
+
+import ai.djl.examples.inference.face.model.FaceDetectedObjects;
+import ai.djl.examples.inference.face.model.Landmark;
+import ai.djl.modality.cv.Image;
+import ai.djl.modality.cv.output.BoundingBox;
+import ai.djl.modality.cv.output.Point;
+import ai.djl.modality.cv.output.Rectangle;
+import ai.djl.ndarray.NDArray;
+import ai.djl.ndarray.NDArrays;
+import ai.djl.ndarray.NDList;
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.DataType;
+import ai.djl.ndarray.types.Shape;
+import ai.djl.translate.Batchifier;
+import ai.djl.translate.Translator;
+import ai.djl.translate.TranslatorContext;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class FaceDetectionTranslator implements Translator<Image, FaceDetectedObjects> {
+    private Batchifier batchifier = Batchifier.STACK;
+    private double confThresh;
+    private double nmsThresh;
+    private int topK;
+    private double[] variance;
+    private int[][] scales;
+    private int[] steps;
+    private int width;
+    private int height;
+
+    public FaceDetectionTranslator(
+            double confThresh,
+            double nmsThresh,
+            double[] variance,
+            int topK,
+            int[][] scales,
+            int[] steps) {
+        this.confThresh = confThresh;
+        this.nmsThresh = nmsThresh;
+        this.variance = variance;
+        this.topK = topK;
+        this.scales = scales;
+        this.steps = steps;
+    }
+
+    @Override
+    public NDList processInput(TranslatorContext ctx, Image input) {
+        width = input.getWidth();
+        height = input.getHeight();
+        NDArray array = input.toNDArray(ctx.getNDManager(), Image.Flag.COLOR);
+        array = array.transpose(2, 0, 1).flip(0); // HWC -> CHW RGB -> BGR
```

Comment:
Do you think if we can replace most of the preprocessing with toTensor?

ToTensor in DJL: https://github.com/awslabs/djl/blob/master/api/src/main/java/ai/djl/ndarray/internal/NDArrayEx.java#L421

Maybe also Normailize method defined in DJL: https://github.com/awslabs/djl/blob/master/api/src/main/java/ai/djl/ndarray/internal/NDArrayEx.java#L411

We had built in Transform function that can apply these transformation: https://javadoc.io/doc/ai.djl/api/latest/ai/djl/modality/cv/transform/package-summary.html

Lastly, is RGB -> BGR really necessary here? In most of the cases the input image is following RGB format